uint8 j
float32[6] pose
key_point[] kps
descriptors des
